protocol/messenger_handler.go

```diff
@@ -293,7 +293,7 @@ func (m *Messenger) createContactRequestNotification(contact *Contact, messageSt
 				// Otherwise, it means we have already passed it to the client, so we add it with a `dismissed` flag
 				// so it can clean up
 				if !found {
-					messageState.Response.AddActivityCenterNotification(notification)
+					messageState.Response.AddActivityCenterNotification(notification) // TODO: does we need here to update state?
```

Comment:
I would leave out TODO comment because it's just an open question :man_shrugging: In case you want to keep it, it's good practice to add your username to the TODO comment, so it's easier for other collaborators to know who they can talk to.

Reply:
This todo for reviewers) I'm not sure we need to update state here because i don't see saving updated notification to the database

Reply:
Cool, I didn't realize that because the comment doesn't say it's just for reviewers. I see a lot of people (including myself) doing this differently @MishkaRogachev. If there's any question, comment, etc, they can be added as a Github comment on the line of the code, Github then creates an open thread that other reviewers can comment on. This is better IMO because you don't need to remember to clean up code comments before merging.